### PR TITLE
Add BLS documentation

### PR DESCRIPTION
**Pull request by nv-braf**

### CODE REVIEW

File: docs/config_search.md [hunk 1]
```diff
@@ -36,13 +37,14 @@ Model Analyzer's `profile` subcommand supports multiple modes when searching to
 - [Brute Force Search](config_search.md#brute-search-mode)
   - **Search type:** Brute-force sweep of the cross product of all possible configurations
   - **Default for:**
-    - Single non-ensemble models
+    - Single non-ensemble/BLS models
```
Comment (tgerdesnv) [inline docs/config_search.md:40]: I read this as single non-ensemble models and single BLS models

File: docs/config_search.md [hunk 2]
```diff
@@ -61,12 +63,15 @@ Model Analyzer's default search mode depends on the type of model and if you are
 - [Ensemble Model Search](config_search.md#ensemble-model-search):
   - **Default Search type:** [Quick Search](config_search.md#quick-search-mode)
   - **Command:** N/A
+- [BLS Model Search](config_search.md#bls-model-search):
+  - **Default Search type:** [Quick Search](config_search.md#quick-search-mode)
+  - **Command:** N/A
```
Comment (tgerdesnv) [inline docs/config_search.md:68]: I don't like that command is listed for all 4 but only one of them actually has that information.

File: docs/config_search.md (hunk 1 above)
Comment (nv-braf) [reply docs/config_search.md]: Fixed

File: docs/config_search.md (hunk 2 above)
Comment (nv-braf) [reply docs/config_search.md]: Removed

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**